### PR TITLE
oa.ttl - Specify W3C Document and Software license

### PR DESCRIPTION
**Pull request by stain**
.. as agreed in #66 

Note that this pull request does **not** yet update all the descriptions and terms according to the latest http://w3c.github.io/web-annotation/model/wd2/

### CODE REVIEW

**Comment by stain** [comment]
You may preview the vocabulary docs by looking at the HTML rendering (running `make`) at http://stain.github.io/web-annotation/vocabulary/wd/

**Comment by stain** [comment]
I assume that also the JSON-LD context should have the Software & Document license then.

**Comment by stain** [comment]
I used a dummy `metadata` block in the `*_frame.jsonld` files to declare their licenses, as JSON-LD has no comments.  As `metadata` does not match anything, it is ignored by the JSON-LD Framing.

Similarly in anno.jsonld the `dc:rights` and `dcterms:license` annotations are straight on the outer object (the file itself), rather than inside `@context`.

**Comment by azaroth42** [comment]
Thanks @stain! 

I'm concerned about the use of a key that might be overridden in another context (e.g. https://github.com/IIIF/iiif.io/blob/master/source/api/presentation/2/context.json#L102 defines a `metadata` key...)

Something less likely to collide seems useful? `__header` ?  Or, given the magic-ness of @ ... `@__header` ?

**Comment by stain** [comment]
A frame specifies it's own @context, so it would not be easily crash,
except of course if the frame is manually modified (which is exactly what
this license permits) - so I will change it to `__header`.

File: vocabulary/wd/index.html [hunk 1]
```diff
@@ -46,17 +52,17 @@ <h1 property="dc:title schema:name">W3C Web Annotation vocabulary</h1>
 <dl><dt>Imported Ontologies:</dt>
 <dd><a property="owl:imports schema:mentions" resource="http://www.w3.org/ns/prov#" href="http://www.w3.org/ns/prov#">PROV-O</a></dd><dd><a property="owl:imports schema:mentions" resource=" http://www.w3.org/2004/02/skos/core" href=" http://www.w3.org/2004/02/skos/core"> SKOS</a></dd></dl>
 
-<dl>This work is licensed under a <a rel="license" href="http://www.w3.org/Consortium/Legal/copyright-documents">Copyright © 2014 W3C® (MIT, ERCIM, Keio, Beihang), All Rights Reserved. W3C liability, trademark and document use rules apply.</a>.</dl>
-<span property="dc:license" resource="http://www.w3.org/Consortium/Legal/copyright-documents"></span>
-<hr/>
+<dl>This work is licensed under a license: <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">Copyright © 2016 W3C ® (MIT, ERCIM, Keio, Beihang). W3C liability, trademark use rules apply. This Document is licensed under the W3C Software and Document license</a>.</dl>
```
Comment (iherman) [inline vocabulary/wd/index.html]: This is incorrect, please re-establish what was there before. The recommendation's license is what it used to be (except of course for the date), and there is no place for the software license at this point

**Comment by iherman** [comment]
Guys, 

this is a bit messy. @azaroth42, you have two repositories now for the vocabulary: 'vocabulary' and 'vocab/wd'. My understanding is that the latter is the latest release to be published, but most of @stain's changes are in the 'vocabulary'. What is happening?

In any case… I glanced at the changes proposed by @stain. Most of them look o.k. to me, except (and I annotated that separately) that the license of the vocabulary Recommendation itself in the status section MUST NOT change. That is against the current pubrules, and the document will not be accepted with what is proposed.

There may be a reference in the document somewhere that the vocabulary itself is available in various syntaxes, and it is all right to add there, if we want, that those files are released under the software license. But it is not strictly necessary, because the licenses are now set in the files themselves.

**Comment by stain** [comment]
Agree, @iherman - so I've updated `widoco.properties` so that the generated `wd.html` now have the document license again.

**Comment by stain** [comment]
Sorry, I got confused and didn't see that vocab/wd already contained a nice html, I thought that folder just had lots of examples and some code.  Is this HTML written by hand?

**Comment by stain** [comment]
I'll leave #188 there then for @azaroth42 to sort it out - feel free to use the updated oa.ttl from vocabulary/ there.

**Comment by iherman** [comment]
Same question as in https://github.com/w3c/web-annotation/pull/188#issuecomment-202334013: where are we with this, what happens to the namespace and context files? 

Cc @azaroth42 @stain

**Comment by azaroth42** [comment]
http://stain.github.io/web-annotation/vocabulary/wd/ seems very out of sync. Is it up to date with the content of the PR?

**Comment by stain** [comment]
Yes, that _is_ out of date as my gh-pages branch basically only did a
software License change, pull request #187

See instead the rendering on

https://rawgit.com/stain/web-annotation/updating/vocabulary/wd/index.html
Which is from
https://github.com/stain/web-annotation/blob/updating/vocabulary/oa.ttl
(pull request #188)
which should be more in sync with the model WD - however I didn't have time
to go through the Specifiers.

As I realise you have the handwritten vocabulary document, this oa.ttl
needs to be aligned with vocab/wd/index.html, and the generated oa.html can
be scrapped (although it is useful to check the oa.ttl is annotated
correctly).

Do you want me to have a go on that?

What should we have as oa.html on the namespace URI http://www.w3.org/ns/oa
for browsers? A redirect straight to the vocab on /TR/ or an intermediate
landing page?

http://stain.github.io/web-annotation/vocabulary/wd/ seems very out of
sync. Is it up to date with the content of the PR?

**Comment by stain** [comment]
I'm closing this in favour of #188

**Comment by azaroth42** [comment]
Thanks Stian! The machine readable versions are essential, so your assistance is greatly appreciated :)

Let's split this:

> What should we have as oa.html on the namespace URI http://www.w3.org/ns/oa
> for browsers? A redirect straight to the vocab on /TR/ or an intermediate
> landing page?

To an issue.

**Pull request closed without merging**